src/main/java/logisticspipes/network/GuiHandler.java

```diff
@@ -76,7 +54,7 @@ public Object getServerGuiElement(int ID, EntityPlayer player, World world, fina
         if (ID < 110 && ID > 0) {
             switch (ID) {
                 case GuiIDs.GUI_FluidSupplier_ID:
-                    if (pipe == null || pipe.pipe == null || !(pipe.pipe instanceof PipeItemsFluidSupplier)) {
+                    if (pipe == null || !(pipe.pipe instanceof PipeItemsFluidSupplier)) {
```

Comment:
redundant checking for null (and cases below - same), because instanceof already checks for null

Reply:
it's not redundant since it's pipe and pipe.null